_data/glossary/secret.yml

```diff
@@ -0,0 +1,14 @@
+id: secret
+name: Secret
+full-link: /docs/concepts/configuration/secret/
+related:
+- pods
+- volume
+tags:
+- core-object
+- security
+short-description: >
+  Stores sensitive information, such as passwords, OAuth tokens, and ssh keys.
+long-description: >
+  Allows for more control over how sensistive information is used and reduces the risk of accidental exposure. 
```

Comment:
s/sensistive/sensitive
Mention encryption. Maybe also compare with ConfigMap? This is a nice resource: https://medium.com/google-cloud/kubernetes-configmaps-and-secrets-68d061f7ab5b
Thanks for your help!

Reply:
done